src/ResizeSensor.js

```diff
@@ -28,6 +28,18 @@
             return window.setTimeout(fn, 20);
         };
 
+    // Passive events for better scroll performance
+    // https://github.com/WICG/EventListenerOptions/blob/gh-pages/explainer.md
+    var passiveEvents = false;
+    try {
+        var opts = Object.defineProperty({}, 'passive', {
+            get: function () {
+                passiveEvents = { passive: true };
+            }
+        });
+        window.addEventListener('test', null, opts);
```

Comment:
Why not just `window.addEventListener('test', null, {passive: true})`?

//edit: Ah, I see. 👍